### DISCUSSION TITLE
Walker waypoints

### DISCUSSION THREAD

**Discussion by catalin-duta** [Keriew/augustus#666] [Q&A]
I just started looking at the walker waypoints feature (from 2.0 I think) with the blue highlights and I have some curiosities:
- are these supposed to take into account roadblocks or gates? It doesn't seem like it on the overlay
- is there any way to know the intersections through which the walkers pass? Some tiles are very close to one another in "gated communities" and I can't figure out the direction in which the walkers go
- what overlay should I use to figure out where the labor seekers go?
- are these "fixed" or can they just change? I've encountered situations where a prefect or a librarian just stopped passing through a road after good years of service, without any changes to the roads in that area. 
- also, should they be calculated by the game in any way based on a building's position? For example, rebuilding a collapsed school in the exact same spot leads to way different coverage
- I think there was a discussion at one point around being able to change these paths, but I can't find it or I don't know what to search for. What's the outlook on that?
- Have gardens always been considered as a road for random walkers? I was under the impression that they are only used for "return trips".

**Comment by catalin-duta** [comment]
![Screenshot_20220901-203716399](https://user-images.githubusercontent.com/93785920/187977792-f085c4fc-b50a-42b7-90ec-0e5d79d6eb2e.jpg)
Here are the walker waypoints for an amphitheater.
Two of them are not even connected by road. What does it mean exactly?

**Reply by crudelios** [reply]
This is an old feature that I think is outdated by now.

It should probably be removed and replaced by something more meaningful, like the actual paths the walkers do make.
